### PR TITLE
Introduce MSetRTL style.

### PR DESCRIPTION
**Pull request by NikitaFil**
Makes rtl layout for the specific part of content.

### CODE REVIEW

File: lib/material/tests/test_mparagraph_aligns.flow [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import material/material;
+import material/material_manager;
+import material/material_dialog;
+
+main() {
+	setLang("ar");
+    manager = makeMaterialManager([]);
+    text = "Science, my boy, is made up of mistakes, but they are mistakes which it is useful to make, because they lead little by little to the truth.";
```
Comment (ykuzmenko73) [inline lib/material/tests/test_mparagraph_aligns.flow]: Could you check how "\n" symbol inside the text will be honored?

File: lib/tropic/tropic_gui.flow [hunk 2]
```diff
@@ -58,16 +58,30 @@ export {
 	TGrid(c : [[Tropic]]) -> Tropic;
 
 	// Arabic(RTL) versions
+	TColsRTL(t : [Tropic], isRTL : bool) -> Tropic;
```
Comment (ykuzmenko73) [inline lib/tropic/tropic_gui.flow]: We don't have M*RTL functions, just M*A, maybe T*RTL should be internal only? AFAIU, the are not intended  to be used outside tropic's implementation

File: lib/material/internal/material_grid.flow [hunk 3]
```diff
@@ -34,6 +34,7 @@ MFlexibleGrid2T(manager : MaterialManager, parent : MFocusGroup, m : MFlexibleGr
 		|> (\st -> arrayPush(st, TEnabled(en)))
 		|> (\st -> arrayPush(st, TLockWidth(lockWidth)))
 		|> (\st -> arrayPush(st, TCropView(cropView)))
+		|> (\st -> arrayPush(st, TSetRTL(par.rtl)))
```
[the remark this answers is not in the document]
Comment (sergiy-stryzhenko) [reply lib/material/internal/material_grid.flow]: Or we could `concat(st, [TEnabled(en), TLockWidth(lockWidth), etc.])`, couldn't we?

File: lib/tropic/tropic_gui.flow (hunk 2 above)
Comment (NikitaFil) [reply lib/tropic/tropic_gui.flow]: T*RTL are used for Material implementation, so they should be exported here. However, I can add a comment about T*A preferability.

File: lib/material/internal/material_grid.flow (hunk 3 above)
[the remark this answers is not in the document]
Comment (NikitaFil) [reply lib/material/internal/material_grid.flow]: For sure, concat / concatA is preferable here.

File: lib/material/tests/test_mparagraph_aligns.flow (hunk 1 above)
Comment (NikitaFil) [reply lib/material/tests/test_mparagraph_aligns.flow]: Tested with \n, works fine. 
image

File: lib/tropic/tropic_gui.flow (hunk 2 above)
Comment (brawaga) [reply lib/tropic/tropic_gui.flow]: All this renaming is bad, hence assumes these structs/functions work only with RTL, whilst they implemented to work for both LTR and RTL cases. Must be turned back.

**Pull request merged**